debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/oracle/generated/PlSqlParser.g4

```diff
@@ -3230,7 +3230,8 @@ end_time_column
     ;
 
 column_definition
-    : column_name (datatype | type_name)
+    : out_of_line_ref_constraint | out_of_line_constraint
```

Comment:
Hi @Naros, I am not entirely sure if this fix would be the right to do. Although the change now recognizes both the constraints as `out_of_line_ref_constraint` it leads to some test failures. What's your opinion on this?

Reply:
This works. Thanks.